### PR TITLE
Add Netgate

### PR DESCRIPTION
**Pull request by nach131**
**Issue:** closes #

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by nach131** [comment]
added Nergate

**Comment by nach131** [comment]
push branch

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7584,6 +7584,12 @@
             "source": "https://brand.netflix.com/en/assets/brand-symbol",
             "guidelines": "https://brand.netflix.com/en/assets/brand-symbol"
         },
+        {
+            "title": "Netgate",
+            "hex": "1475CF",
+            "source": "https://www.netgate.com/",
+            "guidelines": "https://www.netgate.com/"
```
Comment (adamrusted) [inline _data/simple-icons.json:7591]:
```suggestion
            "source": "https://www.netgate.com"
```

Where on this page is the SVG available? I can only see PNGs there currently.

**Comment by service-paradis** [comment]
Closing this one for inactivity.

**Pull request closed without merging**